plasmapy/formulary/braginskii.py

```diff
@@ -1151,19 +1153,18 @@ def _nondim_thermal_conductivity(
             raise ValueError(
                 f"Unrecognized model '{model}' in _nondim_thermal_conductivity"
             )
+    elif model == "braginskii":
+        kappa_hat = _nondim_tc_i_braginskii(hall, field_orientation)
+    elif model == "ji-held":
+        kappa_hat = _nondim_tc_i_ji_held(hall, Z, mu, theta, field_orientation)
+    elif model in ["spitzer-harm", "spitzer"]:
+        raise NotImplementedError(
```

Comment:
Since this line is being modified now, it might be worthwhile to add a test for it.